python/tests/sbp/test_table.py

```diff
@@ -40,7 +40,7 @@ def test_table_count():
   Test number of available messages to deserialize.
 
   """
-  number_of_messages = 204
+  number_of_messages = 205
```

Comment:
I've updated the number of expected messages by the amount I've added in this PR, but the test still expects 204. Am I missing something since this is a new package?

Reply:
Think you need to run the `make all` target to generate the various language bindings for new messages.

Reply:
Yes I did. I didn't commit yet the autogenerted code (to not pollute the PR in case there is something wrong in my yaml)

Reply:
There's a lot of build dependencies so it's probably best to build in the docker image.

Reply:
Correct. I've been using the image indicated in the README. This one `swiftnav/libsbp-build:2021-10-13`.